src/viam/app/data_client.py

```diff
@@ -371,13 +371,21 @@ async def get_latest_tabular_data(
 
         ::
 
-            time_captured, time_synced, payload = await data_client.get_latest_tabular_data(
-                part_id="<PART-ID>",
-                resource_name="<RESOURCE-NAME>",
-                resource_subtype="<RESOURCE-SUBTYPE>",
-                method_name="<METHOD-NAME>"
+            tabular_data = await data_client.get_latest_tabular_data(
+                part_id="77ae3145-7b91-123a-a234-e567cdca8910",
+                resource_name="camera-1",
+                resource_subtype="rdk:component:camera",
+                method_name="GetImage"
             )
 
+            if tabular_data and len(tabular_data) == 3:
```

Comment:
```suggestion
            if tabular_data:
```
(minor) I think the `len` check here is unnecessary, since the return value will always be either `None` or a tuple of length 3.

Reply:
Gotcha, if I run without the check though and it returns `None` I get this error `TypeError: cannot unpack non-iterable NoneType object`. Is that expected/ok?